rust/rbac-registration/src/cardano/cip509/validation.rs

```diff
@@ -399,9 +399,9 @@ pub fn validate_role_data(
 
 /// Checks that there are no unknown roles.
 fn validate_role_numbers<'a>(
```

Comment:
This function not needed anymore, because its not possible to have a `RoleId` instance with the unknown value.
Lets remove it